plugins/provisioners/ansible/cap/guest/debian/ansible_install.rb

```diff
@@ -0,0 +1,28 @@
+
+module VagrantPlugins
+  module Ansible
+    module Cap
+      module Guest
+        module Debian
+          module AnsibleInstall
+
+            def self.ansible_install(machine)
```

Comment:
Note that using Ansible PPA (packaged for Ubuntu releases) is also possible, and allow to easily get the latest Ansible version. Having some more feedbacks from Debian users would be good...